tests/cli/test_datasource_sqlite.py

```diff
@@ -78,6 +78,43 @@ def test_cli_datasource_list(
                 "success": True,
             }
         ),
+        mock.call(
+            {
+                "event_payload": {
+                    "anonymized_name": anonymized_name,
+                    "parent_class": "SqlAlchemyDatasource",
+                },
+                "event": "data_context.add_datasource",
+                "success": True,
+            }
+        ),
+        mock.call(
+            {
+                "event": "datasource.sqlalchemy.connect",
+                "event_payload": {
+                    "anonymized_name": anonymized_name,
+                    "sqlalchemy_dialect": "sqlite",
+                },
+                "success": True,
+            }
+        ),
+        mock.call(
+            {"event_payload": {}, "event": "data_context.__init__", "success": True}
+        ),
+        mock.call(
+            {
+                "event": "cli.datasource.list.begin",
+                "event_payload": {"api_version": "v3"},
+                "success": True,
+            }
+        ),
+        mock.call(
+            {
+                "event": "cli.datasource.list.end",
+                "event_payload": {"api_version": "v3"},
+                "success": True,
+            }
+        ),
```

Comment:
These should have been emitted before. Now emitted because we are explicitly enabling `usage_statistics` for the test at the DataContext-instantiation level.